src/databricks/labs/ucx/azure/access.py

```diff
@@ -512,9 +512,23 @@ def create_access_connectors_for_storage_accounts(self) -> list[tuple[AccessConn
         thread_name = "Creating access connectors for storage accounts"
         results, errors = Threads.gather(thread_name, tasks)
```

Comment:
What does the `.gather` return for result when there is an error?

Reply:
This would return the output of _create_access_connector_for_storage_account function that is invoked from the thread so list of tuple[AccessConnector, str]